packages/authhero/src/components/FormNodePage.tsx

```
@@ -21,7 +21,7 @@ const RichTextComponent = (comp: RichTextComponentProps) => {
   return (
     <div
       className="rich-text mb-6 prose prose-gray max-w-none [&>*:last-child]:mb-0 [&_h1]:mb-6 [&_h2]:mb-6"
-      dangerouslySetInnerHTML={{ __html: comp.config.content }}
+      dangerouslySetInnerHTML={{ __html: comp.config?.content ?? "" }}
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)